src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -0,0 +1,169 @@
+package teammates.ui.controller;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.google.appengine.api.datastore.Text;
+
+import teammates.common.datatransfer.FeedbackSessionType;
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.EmailType;
+import teammates.common.util.Logger;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.TimeHelper;
+
+public abstract class InstructorFeedbackAbstractAction extends Action {
+
+    private static final Logger log = Logger.getLogger();
+
+    /**
+     * Sets the attributes for specified session.
+     * @param isCreatingSession true for newly creating session and false for already created session.
+     * @return session attributes
+     */
+    protected FeedbackSessionAttributes extractFeedbackSessionData(boolean isCreatingSession) {
+        // TODO make this method stateless
+
+        // Null checks for parameters not done as null values do not affect data integrity
+
+        FeedbackSessionAttributes newSession = new FeedbackSessionAttributes();
+        newSession.setCourseId(getRequestParamValue(Const.ParamsNames.COURSE_ID));
+
+        // If the class is InstructorFeedbackAddAction then sanitize the newly given title
+        String title = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        if (isCreatingSession) {
+            title = SanitizationHelper.sanitizeTitle(title);
```

Comment:
@whipermr5 I also thin that we can sanitize title in both cases. As @leeyimin  have not replied yet. Can we proceed with sanitizing both ?